spread.yaml

```diff
@@ -261,9 +266,18 @@ prepare: |
     # (needs to be in spread.yaml directly because there's nothing else on the filesystem yet)
     if [ -f current.delta ]; then
         tf=$(mktemp)
-        # poor-man's "quiet"
-        apt-get update >& "$tf" || ( cat "$tf"; exit 1 )
-        apt-get install -y xdelta3 curl >& "$tf" || ( cat "$tf"; exit 1 )
+        # NOTE: We can't use tests/lib/pkgdb.sh here as it doesn't exist yet
```

Comment:
I was already asked once or twice why I don't use `tests/lib/pkgdb.sh` here. So I will leave the comment to make that visible but improve it a bit.